Test failure: trx_finality_status_test

https://github.com/AntelopeIO/leap/actions/runs/3351941354/jobs/5553755788

```
Traceback (most recent call last):
  File "/__w/leap/leap/build/tests/trx_finality_status_test.py", line 158, in <module>
    validateTrxState(status[0], present=False)
  File "/__w/leap/leap/build/tests/trx_finality_status_test.py", line 155, in validateTrxState
    assert group, \
AssertionError: ERROR: getTransactionStatus shouldnot  contain "block_number", "block_id", or "block_timestamp" since state was "IN_BLOCK".
status: {
 "state": "IN_BLOCK",
 "block_number": 172,
 "block_id": "000000ac4998e8e1168a7f5e15a0e969fdee60afe36f846041a62ee2181789dc",
 "block_timestamp": "2022-10-29T15:26:26.000",
 "expiration": "2022-10-29T15:27:55.000",
 "head_number": 172,
 "head_id": "000000ac4998e8e1168a7f5e15a0e969fdee60afe36f846041a62ee2181789dc",
 "head_timestamp": "2022-10-29T15:26:26.000",
 "irreversible_number": 131,
 "irreversible_id": "00000083d37d401c227e3c704a3026e1bd864f9ec5647e3bcccbd81c37d88c77",
 "irreversible_timestamp": "2022-10-29T15:26:05.500",
 "earliest_tracked_block_id": "00000018aa7395df619cd27563f15d504885de153b206a1837c2f150d43540d6",
 "earliest_tracked_block_number": 24
}
```

--- a/tests/trx_finality_status_test.py
+++ b/tests/trx_finality_status_test.py
@@ -142,7 +142,7 @@ try:
     state = getState(retStatus)
 
     assert (state == localState or state == inBlockState), \
-        f"ERROR: getTransactionStatus didn't return \"{localState}\" state.\n\nstatus: {json.dumps(retStatus, indent=1)}"
+        f"ERROR: getTransactionStatus didn't return \"{localState}\" or \"{inBlockState}\" state.\n\nstatus: {json.dumps(retStatus, indent=1)}"
     status.append(copy.copy(retStatus))
     startingBlockNum=testNode.getInfo()["head_block_num"]
 
@@ -150,12 +150,13 @@ try:
         bnPresent = "block_number" in status
         biPresent = "block_id" in status
         btPresent = "block_timestamp" in status
-        desc = "" if present else "not "
+        desc = "" if present else " not"
         group = bnPresent and biPresent and btPresent if present else not bnPresent and not biPresent and not btPresent
         assert group, \
             f"ERROR: getTransactionStatus should{desc} contain \"block_number\", \"block_id\", or \"block_timestamp\" since state was \"{getState(status)}\".\nstatus: {json.dumps(status, indent=1)}"
 
-    validateTrxState(status[0], present=False)
+    present = True if state == inBlockState else False
+    validateTrxState(status[0], present)
 
     def validate(status, knownTrx=True):
         assert "head_number" in status and "head_id" in status and "head_timestamp" in status, \

